File: patch_codegen_mod.py
with open("crates/codegen-llvm/src/codegen/mod.rs", "r") as f:
    code = f.read()

code = code.replace("""        add(self, "__bs_alloc_array", self.i64_ty.fn_type(&[], false));""", """        add(self, "__bs_alloc_array", self.i64_ty.fn_type(&[], false));
        add(self, "__bs_alloc_owned_array", self.i64_ty.fn_type(&[], false));
        add(self, "__bs_alloc_arena_array", self.i64_ty.fn_type(&[self.i32_ty.into()], false));""")

with open("crates/codegen-llvm/src/codegen/mod.rs", "w") as f:
    f.write(code)

File: patch_codegen_mod.rs
use std::fs;

fn main() {
    let mut content = fs::read_to_string("crates/codegen-llvm/src/codegen/mod.rs").unwrap();
    let old = r#"                (self.i64_ty.const_int(0xFFF9, false), print_closure),
                (self.i64_ty.const_int(0xFFFC, false), print_obj),"#;
    let new = r#"                (self.i64_ty.const_int(0xFFF9, false), print_closure),
                (self.i64_ty.const_int(0xFFF0, false), print_closure),
                (self.i64_ty.const_int(0xFFFC, false), print_obj),
                (self.i64_ty.const_int(0xFFFE, false), print_obj),"#;
    content = content.replace(old, new);
    fs::write("crates/codegen-llvm/src/codegen/mod.rs", content).unwrap();
}
